src/noteeditorlogic.cpp

```diff
@@ -20,6 +20,7 @@ NoteEditorLogic::NoteEditorLogic(CustomDocument *textEdit, QLabel *editorDateLab
                                  DBManager *dbManager, QObject *parent)
     : QObject(parent),
       m_textEdit{ textEdit },
+      m_highlighter{ new CustomMarkdownHighlighter { m_textEdit->document() } },
```

Comment:
```suggestion
      m_highlighter{ new CustomMarkdownHighlighter{ m_textEdit->document() } },
```

Just to make `clang-format` happy.

Reply:
It's not the way I like it, but oh well. :-)